topo_processor/util/conversions.py

```diff
@@ -71,3 +48,22 @@ def nzt_datetime_to_utc_datetime(date: str) -> datetime:
     utc_time = nz_time.astimezone(utc_tz)
 
     return utc_time
+
+
+def quarterdate_to_datetime(value):
```

Comment:
The function name is confusing as the return type is not a `datetime` but a `string`. To avoid that you also type the function.

Reply:
Do not hesitate to create a unit test for this conversion function as well

Reply:
I've updated, is it better now?